up42/auth.py

```diff
@@ -156,10 +212,10 @@ def _get_token(self):
         self.token = token_response["data"]["accessToken"]
 
     def _get_workspace(self) -> None:
-        """Get workspace id belonging to authenticated project."""
-        url = f"https://api.up42.{self.env}/projects/{self.project_id}"
+        """Get user id belonging to authenticated account."""
```

Comment:
if this is not returning workspace id anymore, consider renaming the function.

Reply:
Yes, just FYI workspace_id and user_id are the same value. I will change it.